### PR TITLE
Add Operator Lifecycle Manager (OLM) team

### PR DESCRIPTION
**Pull request by simrandhaliw**
* Adds @openshift/openshift-team-olm to 0000_50_cluster_monitoring_operator_04-config.yaml
* Adds @openshift/openshift-team-olm to data-collection.md

### CODE REVIEW

**Comment by simrandhaliw** [comment]
OLM team relies on these metrics for PromQL queries used in OLM Grafana dashboards
and would like to be notified when they change.
https://github.com/operator-framework/operator-lifecycle-manager

File: Documentation/data-collection.md [hunk 1]
```diff
@@ -41,32 +41,32 @@ data:
     # available. This is used to ensure that updates are being properly
     # served to clusters.
     - '{__name__="cluster_version_available_updates"}'
-    # cluster_operator_up reports the health status of the core cluster
+    # (team-OLM) cluster_operator_up reports the health status of the core cluster
```
Comment (lilic) [inline Documentation/data-collection.md:44]: Can you change it to `@openshift/openshift-team-olm` if you think everyone is in that team who needs to be pinged about any changes.

Comment (simrandhaliw) [reply Documentation/data-collection.md:44]: Does this have to be github handle or can we add slack user group handles? Example: @olmcop

**Pull request merged**